src/lib/Components/App.php

```diff
@@ -83,9 +83,6 @@ public function jsonSerialize()
         return [
             'selector' => self::TAG_NAME,
             'update' => [
-                'attributes' => [
```

Comment:
if you remove that, the title will never be updated (ok it's not completely implemented, see https://jira.ez.no/browse/EZP-27355).
So here, instead, you should update the property `pageTitle` by putting
```php
'update' => [
    'properties' => [
        'pageTitle' => $this->pageTitle
    ],
```